src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+
+const promise1 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was resolved!');
+
+  newDiv.className = 'message';
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  logo.addEventListener('click', () => {
+    resolve(body.insertAdjacentElement('afterbegin', newDiv));
+  });
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was rejected!');
+
+  newDiv.className = 'message error-message';
+
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
+
+promise1.then();
+promise2.then();
```

Comment:
The `then` methods for `promise1` and `promise2` are empty. You should provide callback functions to handle the resolved and rejected states of the promises.